AEPNotificationContent/Sources/Model/TimerPayload.swift

```diff
@@ -0,0 +1,79 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import UIKit
+import UserNotifications
+
+class TimerPayload: Payload {
+    // MARK: - Properties
+
+    /// The image URL for the timer expired view
+    var alternateImageURL: URL
+
+    /// The color of the timer text
+    var timerColor: UIColor
+
+    /// The duration of the timer
+    var expiryTime: TimeInterval
+
+    /// titleBody of non-expired timer template view
+    lazy var titleBodyPayload: TitleBodyPayload = {
+        let title = notificationContent.expandedTitle ?? notificationContent.title
+        let body = expandedBody ?? ""
+        return TitleBodyPayload(title: title, body: body)
```

Comment:
should we just make body optional in `TitleBodyPayload`?

Reply:
The reason for keeping this non-optional is we don't have to play around optionals and unwrap them in the viewController.
Empty string is treated as nil in View class and the body is removed from the UI